composer/algorithms/seq_length_warmup/seq_length_warmup.py

```diff
@@ -304,6 +306,10 @@ def _activate_model(self, state: State, logger: Logger) -> None:
                 for optimizer in state.optimizers:
                     optimizer.zero_grad()
 
+                # Raise error if automicrobatching and num_alloc_retries increased, as thrashing
+                # often leads to throughput slowdown
+                if state.auto_microbatching and _get_num_alloc_retries() > num_alloc_retries:
+                    raise RuntimeError('num_alloc_retries > 1 which causes memory thrashing and throughput decrease')
```

Comment:
`num_alloc_retries > 1` doesn't match with `_get_num_alloc_retries() > num_alloc_retries:`

Reply:
Good catch, should be > 0